components/Tabs/tests/snapshot/TabBarView/MDCTabBarViewSnapshotTests.m

```diff
@@ -137,4 +137,63 @@ - (void)testItemsWithMixedTitlesAndImages {
   [self generateSnapshotAndVerifyForView:self.tabBarView];
 }
 
+- (void)testMultipleLineItemViewToFitInTheSingleScreen {
```

Comment:
Please add a similar test that uses the bar's `intrinsicContentSize` to set the bounds so we can be sure it will return the correct size to fit the items.

Reply:
Added.